data/tests.yml

```diff
@@ -730,6 +730,85 @@
         direction: forward
         designated: motor_vehicle
 
+- description: "cycleway:BACKWARD:lane=advisory oneway=yes oneway:bicycle=no"
+  way_id: 25745877
+  rust: false
+  # separators: false
+  # expect_warnings: true
+  tags:
+    highway: "residential"
+    oneway: "yes"
+    oneway:bicycle: "no"
+    sidewalk: "both"
+    shoulder: "no"
+    cycleway:left: "lane"
+    cycleway:left:lane: "advisory"
+    cycleway:right: "no"
+  driving_side: right
+  road:
+    highway: road
+    lanes:
+      - type: travel
+        designated: foot
+      # - type: separator
+      #   markings:
+      #     - style: kerb
+      - type: travel
+        direction: backward
+        designated: bicycle
+      - type: separator
```

Comment:
Wait, why're some of the separators enabled and some commented out?

Reply:
that one is clear, the schema for kerbs is still up in the air...

reminds me that I need to fix that :P